.node-version

```diff
@@ -0,0 +1 @@
+18.14.0
```

Comment:
this is read by `actions/setup-node` and many other tools https://stackoverflow.com/a/62978089/133327